pom.xml

```diff
@@ -431,7 +432,25 @@
                     </execution>
                 </executions>
             </plugin>
-             <plugin>
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-checkstyle-plugin</artifactId>
+                <version>${maven-checkstyle-plugin.version}</version>
+                <configuration>
+                    <configLocation>src/main/resources/checkstyle_ci.xml</configLocation>
+                    <includeTestSourceDirectory>false</includeTestSourceDirectory>
```

Comment:
Should include includeTestSourceDirectory too

Reply:
Ok, got it.